rust/src/lib.rs

```diff
@@ -15,6 +15,8 @@
  * 02110-1301, USA.
  */
 
+//! Library tree module.
```

Comment:
This does not sound right. `The Suricata Rust Core Library` perhaps? But, it could be misleading. @jasonish wdyt?

Reply:
I thought I changed this to `Suricata Library Module`.

Seems I didn't save the file 🤦🏿‍♂️.

Reply:
This is Rust documentation entry point...  
```
//! Suricata is a network intrusion, prevention and monitoring engine.
//!
//! Suricata is a hybrid C and Rust application. What is found here are the
//! components written in Rust.
```
that could probably be expanded a lot, but that's the basics.

Reply:
So I should use this?

Reply:
Yeah, I think so.

Reply:
OK then.

Since the issue asks for a short description, I'll use the `Suricata Rust core library`.